pandas-stubs/_typing.pyi

```diff
@@ -48,6 +48,8 @@ from pandas.core.dtypes.dtypes import (
 
 from pandas.io.formats.format import EngFormatter
 
+Incomplete: TypeAlias = Any
```

Comment:
I assume that is inspired by typeshed? We haven't used that yet. Open to using it (is `_typeshed.Incomplete` possible?)

Reply:
Yes and yes.

> I assume that is inspired by typeshed?

At typeshed, `Incomplete` is used as a placeholder for missing type annotations. Sometimes it is needed, for example to annotate module-level or class-level variables, and sometimes it is used for documentation purposes. It also helps find where missing annotations are as you can literally grep for "Incomplete" to find them.
Although this is equivalent to `Any` from a type checker's perspective, they signal different intention from the project's point of view.

> is _typeshed.Incomplete possible?

It can be used if you prefer. I defined it in _typing because I noticed you define all your reusable typing symbols there.

Reply:
I'm fine either way: I believe typeshed is already included with type checkers, so it wouldn't be a dependency